### ISSUE TITLE
Shadowsocks CLI fails to connect through GFW on Manjaro Linux

### ISSUE THREAD

**Issue by cyberthal** [jlund/streisand#437]
Location: 
Shenzhen

**Description of Problem:** 

On manjaro xfce linux, run 
`sslocal -c ~/.shadowsocks`
It appears to connect fine.
However, all external browsing returns the below error: 

```
Secure Connection Failed

The connection to www.baidu.com was interrupted while the page was loading.

    The page you are trying to view cannot be shown because the authenticity of the received data could not be verified.
    Please contact the website owners to inform them of this problem.
```

**Context:**

This machine has a host Win 8.1 and a guest Virtualbox Manjaro.  Shadowsocks still connects fine to Streisand on Win 8.1 via GUI. 

This problem manifested recently, between 1 day and 1 week.  Before that, worked fine. 
I was forced to reprovision my Digital Ocean server when GFW cracked down, and rebooted to find shadowsocks broken.

The problem does not just affect the browser when its socks proxy is enabled.  My CLI commands are also affected.  E.g. `proxychains4 wget baidu.com` fails to receive any information, and instead outputs this: 

```
[proxychains] config file found: /home/leo3/.proxychains/proxychains.conf
[proxychains] preloading /usr/lib/libproxychains4.so
--2016-11-10 04:56:19--  http://baidu.com/
Resolving baidu.com (baidu.com)... 224.0.0.1
Connecting to baidu.com (baidu.com)|224.0.0.1|:80... connected.
HTTP request sent, awaiting response... No data received.
Retrying.
```

Is this just improved GFW interdiction, which I need to counter with a better Linux configuration?  Here is my .shadowsocks: 

```
{
    "server":"188.166.219.158",
    "server_port":8530,
    "local_address": "127.0.0.1",
    "local_port":1080,
   "password":"x7HCs3gun6EgciD681MB1vH4EgX0JTiA47PRGgM2QhOIoHHwtm0YeFijPpy40Lfc",
    "method":"aes-256-cfb",
    "auth": true,
    "timeout":300,
    "fast_open": false,
    "workers":1
}
```

I've Googled and fiddled with this to no avail.

Update: I tried OpenConnect and the protocol worked.  
http://www.xiangji.me/2016/06/08/circumventing-the-great-fire-wall-GFW-in-china-with-shadowsocks/

**Comment by denoza** [comment]
I also had a problem connecting SS with CLI. I opened an issue a few days back here https://github.com/jlund/streisand/issues/430. Does the GUI clients work for you?

Recently I have faced problems with my SS traffic being throttled, seems like the GFW is detecting it and blocking the remote IP.

**Comment by cyberthal** [comment]
Windows GUI client works.  

For Manjaro Linux, I am unaware of an easy SS GUI option.

I used a Digital Ocean droplet for 1-3 months before the IP address was blocked by the GFW.  I spun up a new droplet which worked.

I looked for a script to streamline the process of changing the IP address but didn't find one.

**Comment by cpu** [comment]
I'm going to close this issue since it has not seen any activity for some time. If you are having similar problems with Shadowsocks using a newly provisioned server using the latest code from master please open a new issue and we will do our best to help. Thanks!
